### DISCUSSION TITLE
CSS Gradient for fabric.js

### DISCUSSION THREAD

**Discussion by lh-sn** [fabricjs/fabric.js#9360] [Show and tell]
I have created a showcase repo about how to apply CSS gradients to fabric.js.

![Screenshot_2023-09-20-18-36-54_primary](https://github.com/fabricjs/fabric.js/assets/137523466/ec3f39fe-1d39-4644-b9d6-0202ea28957d)

Repo: https://github.com/SnapMotion/fabric-gradient-color

**Comment by asturur** [comment]
Very nice.
Please consider also exporting the function you wrote as a utility so that people can take a css definition and get a fabricJS object gradient.
